package.json

```diff
@@ -34,10 +34,10 @@
     "mdcss": "^1.5.2",
     "mdcss-theme-github": "^2.4.2",
     "mocha": "^8.1.3",
-    "next-standard-version": "^2.1.3",
+    "next-standard-version": "^1.0.1",
```

Comment:
Was there a reason to drop this to an older version?

Reply:
No, and I have no idea how that happened. I shall fix.